src/mediaembed.js

```diff
@@ -37,6 +44,89 @@ export default class MediaEmbed extends Plugin {
 	static get pluginName() {
 		return 'MediaEmbed';
 	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		this._attachAutoEmbedingEvents();
+	}
+
+	/**
+	 * Attach events required for "auto-embeding" feature.
+	 *
+	 * @private
+	 */
+	_attachAutoEmbedingEvents() {
```

Comment:
This is definitely a logic of `MediaEmbedEditing`, not `MediaEmbed`.

This could even be a separate plugin like `AutoMediaEmbed` loaded by default so even without config, users are able to get rid of it by creating a custom build.

Reply:
Yes, it could be moved as a separate plugin.